ls1intum/Artemis
# [Exam] Individual working time for student exams

### Checklist

### Motivation and Context
For disadvantage compensation it is necessary to set the working time individually for each student exam.

### Description
- [x] The working time will be now formatted as `mm:ss`
- [x] I've added a small form to change the working to the student exam detail view
- [x] I've added a corresponding server route
- [x] Not related to this feature: I've renamed the button for the creation of text exercises from *Generate* to *Save*

### Steps for Testing
1. Go to an exam that has at least one student exam
2. Change the working time and save
3. Reload the page and check that the new working time was persisted successfully
4. Check in the student exam overview that the working time is displayed correctly

### Screenshots
![image](https://user-images.githubusercontent.com/15168372/86230870-fdbe9880-bb91-11ea-8e03-e3364601acb8.png)
![image](https://user-images.githubusercontent.com/15168372/86230910-0911c400-bb92-11ea-8f41-010c2b6c08e4.png)
